pkg/types/gcp/platform.go

```diff
@@ -7,6 +7,16 @@ import (
 	"github.com/openshift/installer/pkg/types/dns"
 )
 
+// DNSZone contains the information about a specific DNS public or private zone.
+type DNSZone struct {
+	// ProjectID is the project where the zone resides.
+	// +optional
+	ProjectID string `json:"projectID,omitempty"`
+
+	// Zone is the name of the zone.
+	Zone string `json:"zone"`
```

Comment:
`Zone` stutters, how about `Name`?

```suggestion
	Name string `json:"name"`
```